TACC/protx-dashboard
# hotfix/fix caching issue

## Overview: ##

Fixes caching regression from https://github.com/TACC/protx-dashboard/pull/97.  We know have different names of databases.  Previously we had a constant name which were were using as a "tag" with the caching library (`diskcache`) and removing cache related to that `tag` when running the run_queries.py script.  Instead of doing that, this PR just removes everything from the cache in the class of `--clear-cache`.

## Related Jira tickets: ##

None

## Testing Steps: ##
1. Run `docker exec -it protx python3 scripts/run_queries.py --clear-cache`
2. Ensure that cache is updated
